pkg/util/validation/limits_map.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package validation
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+// LimitsMap is a generic map that can hold either float64 or int64 as values.
```

Comment:
I need to use this 3 times total (1 for notifications, 1 for rules per group and 1 for rule groups by tenant) - I thought a generic was warranted but let me know if not.

Reply:
I'm happy to also move this change to a separate PR if it makes your life easier but thought the refactor w/o context didn't made much sense.